operator/src/main/java/oracle/kubernetes/operator/steps/HttpRequestProcessing.java

```diff
@@ -35,8 +35,13 @@
   }
 
   static HttpAsyncRequestStep createRequestStep(HttpRequest request, HttpResponseStep responseStep) {
+    return createRequestStep(request, responseStep, HTTP_TIMEOUT_SECONDS);
+  }
+
+  static HttpAsyncRequestStep createRequestStep(HttpRequest request, HttpResponseStep responseStep,
+      Long httpTimeoutSeconds) {
     return HttpAsyncRequestStep.create(request, responseStep)
-          .withTimeoutSeconds(HTTP_TIMEOUT_SECONDS);
+          .withTimeoutSeconds(httpTimeoutSeconds);
```

Comment:
Should this be wl-shutdown-secs instead of the http-timeout? plus maybe a few seconds as a "buffer"?

It's normal not to get a response if shutdown is taking a long time...

Reply:
@rjeberhard Any thoughts?

Reply:
The `httpTimeoutSeconds` value passed in comes from the Shutdown.timeoutSeconds value specified and I believe this is the same as the default weblogic graceful shutdown value.  I can add the same fudge factor (10 seconds) used when we delete the pod:

PodHelper.DEFAULT_ADDITIONAL_DELETE_TIME = 10;

Reply:
Thanks. It's good that the http timeout in this case is based on the shutdown timeout.  OTOH the variable name is confusing if that's the case. Why not just directly use the shutdown timeout variable (assuming there is one somewhere)? 

Adding a fudge factor sounds good to me. It'd be great if @russgold and/or @rjeberhard could weigh in here too.

Reply:
The original method specified the constant HTTP_TIMEOUT_SECONDS and I created an overloaded method to be able pass the timeout as a parameter and just ended up keeping the name, hence httpTimeoutSeconds.  I can add the same fudge factor that we use in PodHelper.DeletePodStep if we all think  an extra 10 seconds is reasonable, even though it's used for allowing extra time when connecting via WLST.

Reply:
I added the additional fudge factor to the request timeout, see ShutdownManagedServerStep.getRequestTimeoutSeconds().  We can make modify it if we need to based on everyone's thoughts.  @russgold and @rjeberhard could weigh in here too.